File: AD-HOC/1943/main.c
#include <stdio.h>

int getCategory(int k) {
    int categories[7] = {1, 3, 5, 10, 25, 50, 100};
    for (int i = 0; i < 7; i++)
        if (k <= categories[i]) return categories[i];
}

int main() {
    int k;
    scanf("%d", &k);
    printf("Top %d\n", getCategory(k));
    return 0;
}

File: AD-HOC/1943/Main.java
import java.util.Scanner;

public class Main {

    static int getCategory(int k) {
        for (int category : new int[] { 1, 3, 5, 10, 25, 50, 100 })
            if (k <= category)
                return category;
        return 0;
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int k = scanner.nextInt();
        System.out.printf("Top %d\n", getCategory(k));
        scanner.close();
    }
}
